### ISSUE TITLE
can not load EXR with dwaa and dwab compression

### ISSUE THREAD

**Issue by grovergol** [imageio/imageio#709]
Image reader accepts any type of EXR compression besides dwaa and dwab.
that's the error text:
`Cannot read image file blabla. Unknown compression type in image header.` 
for the code:
`im = imageio.imread(file_path)`

magic.from_file shows:
`OpenEXR image data, version 2, storage: scanline, compression: unknown, dataWindow: (0 0)-(639 479), displayWindow: (0 0)-(639 479), lineOrder: increasing y`

at the same time Nuke can open the image and recognize the dwaa compression.

the file
[https://www.dropbox.com/s/fwxgmx8dvj3losr/exr_dwaa_test.exr?dl=0](https://www.dropbox.com/s/fwxgmx8dvj3losr/exr_dwaa_test.exr?dl=0)

**Comment by FirefoxMetzger** [comment]
I'm not an expert on the FreeImage library, but I think it doesn't support `dwaa` and `dwab` compression. If that is the case, then we currently don't support it either and this would be an enhancement. @almarklein Do you know the details here?

@grovergol One way forward would be for ImageIO to wrap a python or C library that supports dwaa/dwab compression. Do you happen to know one? (python preferred, since that is easier to implement)

**Comment by grovergol** [comment]
Hi @FirefoxMetzger nice to meet you and thanks for the fast reply. The only library I know is OpenimageIO it has c library and python bindings. But it's so damn complicated that I got the first results only after two days of fighting with it to compile on windows x64. Still, I can not use it because 3ds max python (where I need it to be) doesn't load the library saying something is missing. Maybe I'm just to silly to make it work. But imageio works great with no problems anywhere (only it has no dwaa).
[https://openimageio.readthedocs.io/en/latest/index.html](https://openimageio.readthedocs.io/en/latest/index.html)
[https://github.com/OpenImageIO/oiio](https://github.com/OpenImageIO/oiio)

**Comment by almarklein** [comment]
I checked the FreeImage docs, and there is no mentioning of dwaa or dwab. A quick search suggests that we don't emit the error you mentioned (`Unknown compression type in image header`), so it's likely that FreeImage does. Not much we can do about it, sorry!

**Comment by FirefoxMetzger** [comment]
@grovergol Funny that you mention OpenImageIO; I just created an issue for adding it as a backend https://github.com/imageio/imageio/issues/710. 

As @almarklein already said, ImageIO is a wrapper around various libraries that can read images and provides an easy-to-use API for them. If none of our current backends support a particular compression type it becomes challenging, because it means adding a new backend.

I will close this issue since there isn't much we can do directly from our end. If you would like to see support for OpenImageIO, feel free to leave a comment in #710. If you happen to come across another (python) library that supports dwaa/dwab compression and that we can wrap, feel free to create a new issue here and we can see if/how we can add support in ImageIO :)
